src/specialized/aarch64.rs

```diff
@@ -0,0 +1,123 @@
+#[derive(Clone)]
+pub struct State {
+    state: u32,
+}
+
+impl State {
+    pub fn new() -> Option<Self> {
+        // Ideally, detection via ELF aux vector would be used to check
```

Comment:
Hmm, calling `calculate` later without any CPU feature detection can lead to undefined behaviour. Could you use the [is_aarch64_feature_detected](https://doc.rust-lang.org/std/macro.is_aarch64_feature_detected.html) macro here to check for `crc` feature support?

The downside is that the macro in `std` is feature gated via `stdsimd`, but maybe that can be added to the list of features we enable on nightly (see my comment on `lib.rs`).

Reply:
As I said, that won't work on my OS right now. I'll send a PR to stdsimd soon to fix that.